numpy/core/tests/test_cython.py

```diff
@@ -67,6 +67,103 @@ def install_temp(request, tmp_path):
             raise RuntimeError(f'could not parse "{install_log}"')
 
 
+def test_is_integer_object(install_temp):
+    import checks
+
+    assert not checks.is_integer(np.timedelta64(1234))
+    assert not checks.is_integer(np.datetime64(1234, "D"))
+
+    assert not checks.is_integer("a")
+
+    assert not checks.is_float(float("nan"))
+    assert not checks.is_float(np.nan)
```

Comment:
copy-paste error: these two lines should be `is_integer`

Reply:
thanks, updated